### PR TITLE
feat (executorReadfile)

### PR DESCRIPTION
**Pull request by yesnault**

### CODE REVIEW

**Comment by yesnault** [comment]
I'll add md5 and size

**Comment by yesnault** [comment]
I'll add md5 and size

File: executors/readfile/README.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+and return content. Content can be used by another steps of testsuite.
+
+Path can contains a fullpath, a wildcard or a directory:
+
+```
+- path: /a/full/path/file.txt
+- path: afile.txt
+- path: *.yml
+- path: a_directory/
+```
```
Comment (fsamin) [inline executors/readfile/README.md]: we need globbing : https://github.com/mattn/go-zglob

Comment (yesnault) [inline executors/readfile/README.md]: not this https://github.com/runabove/venom/pull/18/files#diff-a57396db6dc8d0930fa39fc20da70641R91 ?

Comment (yesnault) [inline executors/readfile/README.md]: ok, I'll try

Comment (yesnault) [inline executors/readfile/README.md]: added

File: executors/readfile/README.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+  result.executor.executor.path
+  result.content
+  result.contentjson
+  result.size.filename
+  result.md5sum.filename
+  result.modtime.filename
+  result.mod.filename
+```
+
+- result.timeSeconds & result.timeHuman: time for read file
```
Comment (sguiheux) [inline executors/readfile/README.md]: Seconds => seconds   timehumain

Comment (yesnault) [inline executors/readfile/README.md]: fixed

**Pull request merged**